test/OrchardCore.Commerce.Tests.UI/Tests/OrderTests/BehaviorOrderTests.cs

```diff
@@ -137,6 +137,71 @@ public BehaviorOrderTests(ITestOutputHelper testOutputHelper)
                 await context.ClickPublishAsync();
 
                 context.Missing(ByQuantity(1));
+
+                // Boolean attributes should work properly in the Order editor.
+                await context.GoToAdminRelativeUrlAsync("/ContentTypes/AddFieldsTo/Product");
```

Comment:
The `/ContentTypes/AddFieldsTo/` has been used a few times, please create an extension method for it.